core/services/pg/q.go

```diff
@@ -177,16 +181,30 @@ func (q Q) ExecQIter(query string, args ...interface{}) (sql.Result, context.Can
 	return res, cancel, q.withLogError(err)
 }
 func (q Q) ExecQ(query string, args ...interface{}) error {
-	_, cancel, err := q.ExecQIter(query, args...)
-	cancel()
-	return err
+	ctx, cancel := q.Context()
+	defer cancel()
+
+	q.logSqlQuery(query, args...)
+	begin := time.Now()
+	defer q.postSqlLog(ctx, begin)
+
+	_, err := q.Queryer.ExecContext(ctx, query, args...)
+	return q.withLogError(err)
 }
 func (q Q) ExecQNamed(query string, arg interface{}) (err error) {
 	query, args, err := q.BindNamed(query, arg)
 	if err != nil {
 		return errors.Wrap(err, "error binding arg")
 	}
-	return q.ExecQ(query, args...)
+	ctx, cancel := q.Context()
+	defer cancel()
+
+	q.logSqlQuery(query, args...)
+	begin := time.Now()
+	defer q.postSqlLog(ctx, begin)
+
+	_, err = q.Queryer.ExecContext(ctx, query, args...)
+	return q.withLogError(err)
```

Comment:
Inlining these blocks so that the log helpers don't need to skip extra extra levels.